lib/styles/app_styles.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+
+import 'app_colors.dart';
+
+class AppTextStyles {
+  static const Color textColorBlack = AppColors.textColorBlack;
+  static const Color textColorWhite = AppColors.textColorWhite;
+  static const double defaultOpacity = 1;
+
+  static TextStyle title(
```

Comment:
format whole file

Reply:
Did you format file? You are missing commas at the end of each  last named param in constructors